### PR TITLE
March news

### PR DESCRIPTION
**Pull request by maelle**
TODO

- [x] Add reminder of the ISC call for proposals in the package dev corner
- [x] Add paragraph to HQ
- [x] Remove tokenizers
- [x] Merge #746 
- [x] Merge events PR

### CODE REVIEW

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+A highlight is that our guide is now bilingual (English and [Spanish](https://devguide.ropensci.org/es/index.es.html)), thanks to work by [Yanina Bellini Saibene](/author/yanina-bellini-saibene/), [Elio Campitelli](/author/elio-campitelli/) and [Pao Corrales](/author/pao-corrales/), and thanks to [support of the Chan Zuckerberg Initiative, NumFOCUS, and the R Consortium](/blog/2023/01/12/multilingual-publishing-en/).
+[Read the guide in Spanish](https://devguide.ropensci.org/es/index.es.html).
+
+Our guide is now also getting translated to [Portuguese](/blog/2023/10/20/news-october-2023/#help-us-translate-our-dev-guide-to-portuguese) thanks to volunteers.
+We are very grateful for their work!
+
+Read more in the [blog post about the release](/blog/2024/03/11/devguide-0.9.0/).
+Thanks to all contributors who made this release possible.
+
+### Interview of our community manager Yanina Bellini Saibene
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:60]:
```suggestion
### Interview with Code for Thought podcast
```

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,667 @@
...
+
+Yanina talked with Peter Schmidt of the [Code for thought podcast](https://codeforthought.buzzsprout.com/1326658/14660478-en-crossing-the-language-barrier-yanina-saibene), about the importance to make computing materials accessible to non-English speaking learners.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:62]:
```suggestion
Our community manager, Yanina Bellini Saibene, talked with Peter Schmidt of the [Code for Thought podcast](https://codeforthought.buzzsprout.com/1326658/14660478-en-crossing-the-language-barrier-yanina-saibene), about the importance of making computing materials accessible to non-English speaking learners. 
```

File: content/blog/2024-03-22-news-march-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+
+## Software :package:
+
+### New packages
+
+
+
+
+The following  package recently became a part of our software suite, or were recently reviewed again:
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.md:66]: I think the code that creates this statement needs to be fixed for single packages the second part too :
```suggestion
The following  package recently became a part of our software suite, or was recently reviewed again:
```

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,667 @@
...
+
+[internetarchive](https://docs.ropensci.org/internetarchive/), an API Client for the Internet Archive. [Issue for volunteering](https://github.com/ropensci/internetarchive/issues/17).
+
+[historydata](https://docs.ropensci.org/historydata/), datasets for historians. [Issue for volunteering](https://github.com/ropensci/historydata/issues/23).
+
+[textreuse](https://docs.ropensci.org/textreuse/), detect text reuse and document similarity. [Issue for volunteering](https://github.com/ropensci/textreuse/issues/97).
+
+[tokenizers](https://docs.ropensci.org/tokenizers/), fast, consistent tokenization of natural language text. [Issue for volunteering](https://github.com/ropensci/tokenizers/issues/86).
+
+[USAboundaries](https://docs.ropensci.org/USAboundaries/) (and USAboundariesdata), historical and contemporary boundaries of the United States of America . [Issue for volunteering](https://github.com/ropensci/USAboundaries/issues/50).
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:610]: Bulleted list?

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,667 @@
...
+
+### Calls for contributions
+
+[Help make qualtRics better! Code contributions wanted](/blog/2024/03/01/qualtrics-call-help/)
+
+[Help make assertr better! Come close issues](/blog/2024/02/27/assertr-call-help/)
+
+[Help users and developers of targets by answering questions!](/blog/2024/02/29/targets-call-help/)
+
+[Help make waywiser better! User requests wanted](/blog/2024/02/19/waywiser-call-help/)
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:620]: Bulleted list?

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,667 @@
...
+
+Also refer to our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.
+
+## Package development corner
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:624]: Okay, I'm officially creating a News Tips note in my Workflowy to keep track of all the awesome tips I get from this section, thank you!

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,667 @@
...
+The [`.Rbuildignore` file](https://blog.r-hub.io/2020/05/20/rbuildignore/) lists the files to not be included when building your package, such as your pkgdown configuration file.
+Trevor L. Davis posted a neat idea on [Mastodon](https://mastodon.social/deck/@trevorld@fosstodon.org/112012986709303927): using a CRAN-specific `.Rbuildignore`, so that CRAN submissions omit some tests and vignettes to keep the package under the size limit.
+
+Regarding tests themselves, remember you can [skip some or all](https://books.ropensci.org/http-testing/cran-preparedness.html#skipping-a-few-tests-on-cran) on CRAN (but make sure you're running them on [continuous integration](https://devguide.ropensci.org/pkg_ci.html)!).
+
+
+### Key advantages of using the keyring package
+
+If your package needs the user to provide secrets, like API tokens, to work, you might be interested in wrapping or recommending the keyring package (maintained by Gábor Csárdi), that accesses the system credential store from R.
+See this recent [R-hub blog post](/2024/02/28/key-advantages-of-using-keyring/).
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:655]:
```suggestion
See this recent [R-hub blog post](https://blog.r-hub.io/2024/02/28/key-advantages-of-using-keyring/).
```

File: content/blog/2024-03-22-news-march-2024/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,667 @@
+---
+title: 'rOpenSci News Digest, March 2024'
+author: 'The rOpenSci Team'
+date: '2024-03-22'
```
Comment (steffilazerte) [inline content/blog/2024-03-22-news-march-2024/index.Rmd:4]: Just a reminder to update the dates here and in the folder 😁

**Review by steffilazerte — COMMENTED** [review]
Looks good @maelle!

Absolutely love the dev corner, as always 👩🏼‍🎓 

I added the coworking information here and to the PR #747, thanks in advance for merging that in for me tomorrow. 

* [ ] publication date is ok -> Update to Friday
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2024-03-22-news-march-2024/index.Rmd (hunk 6 above)
Comment (maelle) [reply content/blog/2024-03-22-news-march-2024/index.Rmd:624]: :tada:

**Pull request merged**